drivers/sensor/realtek/rts5912/Kconfig

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2025, Realtek, SIBG-SD7
+# SPDX-License-Identifier: Apache-2.0
+
+config TACH_RTS5912
```

Comment:
`TACH_` is not necessary here, better to prefix with `REALTEK_`

Reply:
I think I'm starting to see, this is a tachometer that's a part of the rts5912 eval board. Lets keep the discussion at the compatible string comment and take it from there.